projects/js-packages/charts/tsconfig.json

```diff
@@ -1,9 +1,19 @@
 {
 	"extends": "jetpack-js-tools/tsconfig.base.json",
 	"compilerOptions": {
-		"typeRoots": [ "./node_modules/@types/", "src/*" ]
+		"composite": true,
+		"target": "esnext",
+		"module": "esnext",
+		"moduleResolution": "bundler",
+		"sourceMap": true,
+		"declaration": true,
+		"declarationMap": true,
```

Comment:
Turned on declarationMap so that VSCode could follow to the source. Super handy!!